packages/lodestar/test/unit/api/impl/node/node.test.ts

```diff
@@ -94,24 +94,51 @@ describe("node api implementation", function () {
       } as LibP2pConnection);
       const peers = await api.getPeers();
       expect(peers.length).to.equal(2);
-      expect(peers[0].address).not.empty;
+      expect(peers[0].lastSeenP2pAddress).not.empty;
       expect(peers[0].peerId).not.empty;
       // expect(peers[0].enr).not.empty;
       expect(peers[0].direction).to.equal("outbound");
       expect(peers[0].state).to.equal("connected");
-      expect(peers[1].address).not.empty;
+      expect(peers[1].lastSeenP2pAddress).not.empty;
       expect(peers[1].peerId).not.empty;
       // expect(peers[1].enr).not.empty;
       expect(peers[1].direction).to.equal("inbound");
       expect(peers[1].state).to.equal("disconnecting");
     });
+
+    it("should return disconnected peers", async function () {
+      const peer1 = await createPeerId();
+      const peer2 = await createPeerId();
+      networkStub.getAllPeers.returns([generatePeer(peer1), generatePeer(peer2)]);
+      // peer 1 is in connection manager with status "closed"
+      networkStub.getPeerConnection.onFirstCall().returns({
+        remoteAddr: new Multiaddr(),
+        stat: {
+          status: "closed",
+          direction: "outbound",
+        },
+      } as LibP2pConnection);
+      // peer2 is not in connection manager
+      const peers = await api.getPeers();
+      expect(peers.length).to.equal(2);
+      expect(peers[0].lastSeenP2pAddress).not.empty;
+      expect(peers[0].peerId).not.empty;
+      // expect(peers[0].enr).not.empty;
+      expect(peers[0].direction).to.equal("outbound");
+      expect(peers[0].state).to.equal("disconnected");
+      expect(peers[1].lastSeenP2pAddress).to.be.empty;
+      expect(peers[1].peerId).not.empty;
+      // expect(peers[1].enr).not.empty;
+      expect(peers[1].direction).to.equal("NA");
+      expect(peers[1].state).to.equal("disconnected");
```

Comment:
Probably it's not in the scope of this PR but it would be much more helpful to do
```
expect(toPeerSummary(peers[0]).to.deep.equal({
  direction: "outbound",
  state: "disconnected",
  hasPeerId: true,
  hasP2pAddress: true
})
```
Otherwise, on failure you have a message like
```
expected null to not be empty
```
that provides no help at all on knowing the actual issue

Reply:
Or do
```ts
function assertPeer(data) {
  expect(peers[0].lastSeenP2pAddress, `peer ${id} has not lastSeenP2pAddress`).not.empty;
  expect(peers[0].peerId, `peer ${id} has no peerId`).not.empty;
  ....
}
```